# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse/Cargo.toml
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 third_party/rust/mp4parse/Cargo.toml
--- a/third_party/rust/mp4parse/Cargo.toml	Fri Jan 27 19:06:16 2023 +0000
+++ b/third_party/rust/mp4parse/Cargo.toml	Fri Jan 27 21:11:42 2023 +0200
@@ -1,6 +1,6 @@
 [package]
 name = "mp4parse"
-version = "0.16.0"
+version = "0.13.0"
 authors = [
   "Ralph Giles <giles@mozilla.com>",
   "Matthew Gregan <kinetik@flim.org>",
@@ -13,7 +13,7 @@
 documentation = "https://docs.rs/mp4parse/"
 license = "MPL-2.0"
 categories = ["multimedia::video"]
-edition = "2018"
+
 repository = "https://github.com/mozilla/mp4parse-rust"
 
 # Avoid complaints about trying to package test files.
@@ -28,6 +28,7 @@
 [dependencies]
 byteorder = "1.2.1"
 bitreader = { version = "0.3.2" }
+env_logger = "0.8"
 fallible_collections = { version = "0.4", features = ["std_io"] }
 num-traits = "0.2.14"
 log = "0.4"